### PR TITLE
The oleron did not actually have a zero entry despite HINGE = 0

### PR DESCRIPTION
**Pull request by PaulWessel**
We require a z == 0 slice for HINGE == 0 to work.  Fixes issue reported in the forum: https://forum.generic-mapping-tools.org/t/setting-a-color-hinge-for-oleron-color-palette-table-cpt/74

### CODE REVIEW

File: share/cpt/berlin.cpt [hunk 1]
```diff
@@ -140,7 +140,8 @@
 -0.027450	21/14/14	-0.019608	22/14/13
 -0.019608	22/14/13	-0.011764	23/13/11
 -0.011764	23/13/11	-0.003922	24/12/10
--0.003922	24/12/10	0.003922	25/12/9
+-0.003922	24/12/10	0		25/12/9
+0		24/12/10	0.003922	25/12/9
```
Comment (seisman) [inline share/cpt/berlin.cpt]: Are you sure this is correct?

Comment (PaulWessel) [reply share/cpt/berlin.cpt]: Sorry, fixed

File: share/cpt/berlin.cpt [hunk 2]
```diff
@@ -140,7 +140,8 @@
 -0.027450	21/14/14	-0.019608	22/14/13
 -0.019608	22/14/13	-0.011764	23/13/11
 -0.011764	23/13/11	-0.003922	24/12/10
--0.003922	24/12/10	0.003922	25/12/9
+-0.003922	24/12/10	0		25/12/9
```
Comment (seisman) [inline share/cpt/berlin.cpt]: The change still doesn't make sense to me.

The original z-slice is `-0.003922	24/12/10	0.003922	25/12/9`. The color of z=0 should be an interpolated color of `24/12/10` and `25/12/9`, right?

File: share/cpt/cork.cpt [hunk 3]
```diff
@@ -141,7 +141,8 @@
 -0.019608	229/237/238	-0.011764	230/238/238
 -0.011764	230/238/238	-0.003922	230/238/237
 -0.003922	230/238/237	0.003922	231/239/237
```
Comment (seisman) [inline share/cpt/cork.cpt]:
```suggestion
```

**Comment by PaulWessel** [comment]
Give me an hour, driving home...

Paul Wessel, Professor and Chair
Dept. of Earth Sciences (formerly Geology & Geophysics)
SOEST, U of Hawaii at Manoa

**Comment by seisman** [comment]
I'm also very confused about hinge. From what I understand, hinge is only needed when a CPT has a color discontinuity at z=0 (i.e. CPT for earth relief). Of the 7 CPTs changed in this PR, only oleron is discontinuous at z=0, while all others are continuous.

**Comment by PaulWessel** [comment]
Continuous yes, but note that they (berlin, broc, cork, lisbon, roma, tofino) all goes through either a near-white or near-black middle value with either lighter/darker on each side or different shades of two different colors.  These are all meant to have a hinge even though there is no jump at zero.

File: share/cpt/berlin.cpt (hunk 2 above)
Comment (PaulWessel) [reply share/cpt/berlin.cpt]: Rounded to nearest integer there is no difference.

File: share/cpt/cork.cpt (hunk 3 above)
Comment (seisman) [reply share/cpt/cork.cpt]: This change is incorrect. There are duplicated records for `-0.003922` to `0.003922`.

File: share/cpt/vik.cpt [hunk 4]
```diff
@@ -140,7 +140,8 @@
 -0.027450	230/236/237	-0.019608	231/237/236
 -0.019608	231/237/236	-0.011764	233/237/235
 -0.011764	233/237/235	-0.003922	234/237/234
--0.003922	234/237/234	0.003922	235/237/233
+-0.003922	234/237/234	0		235/237/233
+0		234/237/233	0.003922	235/237/233
```
Comment (seisman) [inline share/cpt/vik.cpt]:
```suggestion
0		235/237/233	0.003922	235/237/233
```

**Pull request merged**